networks/osmosis-1/upgrades/v11/testnet/guide.md

```diff
@@ -0,0 +1,120 @@
+# v10 to v11 Testnet Upgrade Guide
+
+Osmosis v11 Gov Prop: <https://testnet.mintscan.io/osmosis-testnet/proposals/56843>
```

Comment:
nit: link not found but I think I saw a message to cosmostation, problem on their end?

Reply:
Yeah, prop spam crashed it 😔